### ISSUE TITLE
Acceptance Criteria - Pass 1 - LR

### ISSUE THREAD

**Issue by djlarj** [Christopher-Green424/websiteQAExample1#29]
## Environment
MacBook Pro (16-inch, 2019)
macOS Big Sur Version 11.5.2
2.6 GHz 6-Core Intel Core i7
Chrome Version 93.0.4577.82 (Official Build) (x86_64)

## Acceptance Criteria
1. The website should comply with WCAG 2.1 AA ADA standards.
 - [ ] - Check if passed.
 Notes: The WAVE Evaluation Tool found 2 contrast errors specifically on the phone number and email address in the footer. Visually, I don't see an issue with this. However, it makes me question why the tool did not flag the "We need volunteers!" link in the header. That text is harder to read. Doing a manual test of the contrast in these areas is difficult due to the background being an image of wood which holds many different hues of colors. While some hues of color may be ADA compliant, other are not. I am a bit lost on this one. 
The Return to top link is not easy to read. 
2. There is a navbar and footer.
  - [ ] - Check if passed.
  Notes: The Nav Bar is not obvious. Visually, one could conclude there is no Nav Bar but if you look at the code, there is a nav element. The footer, however, is obvious and does display well. 
3. All of the links on the website function as intended.
  - [ ] - Check if passed.
  Notes: The Instagram icon in the footer has a broken link. The Logo on the Contact page also has a broken link. Both cases take the user to a 404 File not found error. 
4. The logo of the club is present and readable.
  - [ ] - Check if passed.
  Notes: The logo of the club is present in both the home page and the Contact page but I can see some testers making an argument that the "Established 1993" wording is hard to read. 
5. The website's footer has at least four working social media links.
  - [ ] - Check if passed.
  Notes: The website has 4 social media icons, but the Instagram icon does not have a working link. The other 3 do. This is consistent with both the home page and the Contact page. 
6. The website has a consistent theme between pages.
  -  [x] - Check if passed.
  Notes: Yes
7. The website has at least 5 bordered images on it.
  - [ ] - Check if passed.
  Notes: This is an edge case. The images themself do not have their own borders. However, they happen to be wrapped inside a figure element that does have borders.
8. The website has a contact page with a form and Google Maps embed.
  - [x] - Check if passed.
  Notes: Yes
9. All resources used in the creation of the website are correctly cited and available for use.
  - [ ] - Check if passed.
  Notes: The source for the eagle and the bobcats images are not cited correctly. They are transposed.
